sdk/storage/azure-storage-queue/azure/storage/queue/_message_encoding.py

```diff
@@ -122,23 +140,23 @@ class BinaryBase64DecodePolicy(MessageDecodePolicy):
     is not valid base 64, a DecodeError will be raised.
     """
 
-    def decode(self, content, response):
+    def decode(self, content: str, response: "PipelineResponse") -> bytes:
         response = response.http_response
         try:
             return b64decode(content.encode('utf-8'))
         except (ValueError, TypeError) as error:
             # ValueError for Python 3, TypeError for Python 2
             raise DecodeError(
                 message="Message content is not valid base 64.",
-                response=response,
+                response=response, #type: ignore
```

Comment:
`azure\storage\queue\_message_encoding.py:151: error: Argument "response" to "DecodeError" has incompatible type "PipelineResponse[Any, Any]"; expected "Optional[_HttpResponseCommonAPI]"  [arg-type]`